When updating doplarr it autofills "not_set_by_default" in JAVA_OPTS env variable causing container to throw an error and not load the bot.

[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

<!--- If you are new to Docker or this application our issue tracker is **ONLY** used for reporting bugs or requesting features. Please use [our discord server](https://discord.gg/YWrKVTn) for general support. --->

<!--- Provide a general summary of the bug in the Title above -->
When updating doplarr it autofills "not_set_by_default" in JAVA_OPTS env variable causing container to throw an error and not load the bot. deleting the environmental variable or removing the "not_set_by_default" line does not work because each update it returns.
------------------------------

## Expected Behavior
<!--- Tell us what should happen -->
the environmental variable should either not be in the template or it should default to an empty box so it doesnt fail to load the container.
## Current Behavior
<!--- Tell us what happens instead of the expected behavior -->
i update and a working container breaks until i manually delete the environmental variable or remove the "not_set_by_default" line 
## Steps to Reproduce
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
1.should not be necessary as information above is enough, but here it goes:
2.configure container and remove "not_set_by_default" or delete the JAVA_OPTS env variable, then container works.
3.update container and it no longer works because the env variable with the "not_set_by_default" line is back.
4.

## Environment
**OS:**
**CPU architecture:** x86_64/arm32/arm64
**How docker service was installed:**
<!--- ie. from the official docker repo, from the distro repo, nas OS provided, etc. -->
<!--- Providing context helps us come up with a solution that is most useful in the real world -->
unraid 6.9.2 installed from CA
## Command used to create docker container (run/create/compose/screenshot)
<!--- Provide your docker create/run command or compose yaml snippet, or a screenshot of settings if using a gui to create the container -->

## Docker logs
<!--- Provide a full docker log, output of "docker logs doplarr" -->

Caused by: java.lang.ClassNotFoundException: not_set_by_default
Error: Could not find or load main class not_set_by_default
Caused by: java.lang.ClassNotFoundException: not_set_by_default
Error: Could not find or load main class not_set_by_default
Caused by: java.lang.ClassNotFoundException: not_set_by_default
Error: Could not find or load main class not_set_by_default

this over and over and over^

--- a/readme-vars.yml
+++ b/readme-vars.yml
@@ -26,16 +26,16 @@ param_container_name: "{{ project_name }}"
 param_usage_include_env: true
 param_env_vars:
   - { env_var: "TZ", env_value: "America/New_York", desc: "Specify a timezone to use EG America/New_York" }
-  - { env_var: "DISCORD__TOKEN", env_value: "<discord__token>", desc: "Specify your discord bot token." }
-  - { env_var: "OVERSEERR__API", env_value: "<overseerr__api>", desc: "Specify your Overseerr API key. Leave blank if using Radarr/Sonarr." }
+  - { env_var: "DISCORD__TOKEN", env_value: "", desc: "Specify your discord bot token." }
+  - { env_var: "OVERSEERR__API", env_value: "", desc: "Specify your Overseerr API key. Leave blank if using Radarr/Sonarr." }
   - { env_var: "OVERSEERR__URL", env_value: "http://localhost:5055", desc: "Specify your Overseerr URL. Leave blank if using Radarr/Sonarr." }
-  - { env_var: "RADARR__API", env_value: "<radarr__api>", desc: "Specify your Radarr API key. Leave blank if using Overseerr." }
+  - { env_var: "RADARR__API", env_value: "", desc: "Specify your Radarr API key. Leave blank if using Overseerr." }
   - { env_var: "RADARR__URL", env_value: "http://localhost:7878", desc: "Specify your Radarr URL. Leave blank if using Overseerr." }
-  - { env_var: "SONARR__API", env_value: "<sonarr__api>", desc: "Specify your Sonarr API key. Leave blank if using Overseerr." }
+  - { env_var: "SONARR__API", env_value: "", desc: "Specify your Sonarr API key. Leave blank if using Overseerr." }
   - { env_var: "SONARR__URL", env_value: "http://localhost:8989", desc: "Specify your Sonarr URL. Leave blank if using Overseerr." }
 param_usage_include_vols: true
 param_volumes:
-  - { vol_path: "/config", vol_host_path: "</path/to/appdata/config>", desc: "Contains all relevant configuration files." }
+  - { vol_path: "/config", vol_host_path: "/path/to/appdata/config", desc: "Contains all relevant configuration files." }
 param_usage_include_ports: false
 param_device_map: false
 cap_add_param: false
@@ -45,15 +45,15 @@ opt_param_usage_include_env: true
 opt_param_env_vars:
   - { env_var: "DISCORD__MAX_RESULTS", env_value: "25", desc: "Sets the maximum size of the search results selection" }
   - { env_var: "DISCORD__REQUESTED_MSG_STYLE", env_value: ":plain", desc: "Sets the style of the request alert message. One of `:plain` `:embed` `:none`" }
-  - { env_var: "SONARR__QUALITY_PROFILE", env_value: "<not_set_by_default>", desc: "The name of the quality profile to use by default for Sonarr" }
-  - { env_var: "RADARR__QUALITY_PROFILE", env_value: "<not_set_by_default>", desc: "The name of the quality profile to use by default for Radarr" }
-  - { env_var: "SONARR__ROOTFOLDER", env_value: "<not_set_by_default>", desc: "The root folder to use by default for Sonarr" }
-  - { env_var: "RADARR__ROOTFOLDER", env_value: "<not_set_by_default>", desc: "The root folder to use by default for Radarr" }
-  - { env_var: "SONARR__LANGUAGE_PROFILE", env_value: "<not_set_by_default>", desc: "The name of the language profile to use by default for Sonarr" }
-  - { env_var: "OVERSEERR__DEFAULT_ID", env_value: "<not_set_by_default>", desc: "The Overseerr user id to use by default if there is no associated discord account for the requester" }
+  - { env_var: "SONARR__QUALITY_PROFILE", env_value: "", desc: "The name of the quality profile to use by default for Sonarr" }
+  - { env_var: "RADARR__QUALITY_PROFILE", env_value: "", desc: "The name of the quality profile to use by default for Radarr" }
+  - { env_var: "SONARR__ROOTFOLDER", env_value: "", desc: "The root folder to use by default for Sonarr" }
+  - { env_var: "RADARR__ROOTFOLDER", env_value: "", desc: "The root folder to use by default for Radarr" }
+  - { env_var: "SONARR__LANGUAGE_PROFILE", env_value: "", desc: "The name of the language profile to use by default for Sonarr" }
+  - { env_var: "OVERSEERR__DEFAULT_ID", env_value: "", desc: "The Overseerr user id to use by default if there is no associated discord account for the requester" }
   - { env_var: "PARTIAL_SEASONS", env_value: "true", desc: "Sets whether users can request partial seasons." }
   - { env_var: "LOG_LEVEL", env_value: ":info", desc: "The log level for the logging backend. This can be changed for debugging purposes. One of trace `:debug` `:info` `:warn` `:error` `:fatal` `:report`" }
-  - { env_var: "JAVA_OPTS", env_value: "<not_set_by_default>", desc: "For passing additional java options." }
+  - { env_var: "JAVA_OPTS", env_value: "", desc: "For passing additional java options." }
 
 opt_param_usage_include_vols: false
 opt_param_usage_include_ports: false

